Cringe.Web/Pages/Beatmaps/Index.cshtml

```diff
@@ -112,7 +112,7 @@
                     @Html.DisplayFor(modelItem => item.Rank)
                 </td>
                 <td>
-                    @Html.DisplayFor(modelItem => item.Score)
+                    @Html.DisplayFor(modelItem => item.ScoreV4)
```

Comment:
Don't we want to show normal score on the website regardless?

Reply:
fixed